apps/sr-frontend/src/app/Character.tsx

```diff
@@ -55,6 +60,115 @@ export default function Character() {
                 })
               }
             />
+            <Grid container>
+              <Grid item>
+                <DropdownButton
+                  title={`Eidolon Lv. ${character?.eidolon}`}
```

Comment:
```suggestion
                  title={`Eidolon Lv. ${character?.eidolon ?? 0}`}
```
add def value to not show undefined
![image](https://github.com/frzyc/genshin-optimizer/assets/1754901/35f6e2fe-5734-44f1-b210-5aeeace9f6f7)